lib/plausible/site/admin.ex

```diff
@@ -117,11 +128,16 @@ defmodule Plausible.SiteAdmin do
     Timex.format!(date, "{Mshort} {D}, {YYYY}")
   end
 
-  defp get_owner_email(site) do
+  defp get_owner(site) do
     owner = Enum.find(site.memberships, fn m -> m.role == :owner end)
```

Comment:
If we add `:owner` to preloads at https://github.com/plausible/analytics/pull/3641/files#diff-ffff0c95718b914d284cc320ae6be63fe3da9bddcab1d81abaf0f13f7b8aece5R17, we could change that to

```suggestion
    owner = site.owner
```

Reply:
https://github.com/plausible/analytics/pull/3641/commits/ca390f08842c94276522831ede13f9bf68e818c3

I checked and there's no owner-less sites anymore